docs/topic-guides/blendable-web-apps/html-views-composition-guidelines.md

```diff
@@ -206,5 +206,5 @@ Regarding styling, there are few ways to make the application easier to visually
 
 ## Additional Resources
 
-To find more information about creating HTML View definitions, take a look at [the article linked above](https://starcounter.io/layout-compositions-html-partials/) and the [People app](https://github.com/StarcounterSamples/People/tree/develop/src/People/wwwroot/People/viewmodels) which fully adheres to these guidelines.
+To find more information about creating HTML View definitions, take a look at [the article linked above](https://starcounter.io/layout-compositions-html-partials/) and the [People app](https://github.com/Starcounter/People) which fully adheres to these guidelines.
```

Comment:
This link is less specific than the previous one. Is that intentional?

Reply:
Well it is intentional but wrong 😄 I misread the context. Fixed now